lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -649,6 +655,17 @@ SPIRVFunction *LLVMToSPIRVBase::transFunctionDecl(Function *F) {
       if (LocID >= 0)
         BA->addDecorate(DecorationBufferLocationINTEL, LocID);
     }
+    if (RuntimeAligned && I->getType()->isPointerTy()) {
+      // Order of integer numbers in MD node follows the order of function
+      // parameters on which we shall attach the appropriate decoration. Add
+      // decoration only if MD value is 1.
+      int LocID = 0;
+      if (!isa<MDString>(RuntimeAligned->getOperand(ArgNo)) &&
+          !isa<MDNode>(RuntimeAligned->getOperand(ArgNo)))
+        LocID = getMDOperandAsInt(RuntimeAligned, ArgNo);
+      if (LocID > 0)
```

Comment:
```suggestion
      if (LocID == 1)
```
...seems more cohesive with the above comment:
> Add decoration only if MD value is 1